src/Sylius/Component/Core/Dashboard/SalesDataProvider.php

```diff
@@ -24,40 +25,57 @@ final class SalesDataProvider implements SalesDataProviderInterface
     /** @var EntityManagerInterface */
     private $entityManager;
 
-    public function __construct(EntityManagerInterface $entityManager)
+    /** @var OrderRepositoryInterface */
+    private $orderRepository;
+
+    public function __construct(EntityManagerInterface $entityManager, OrderRepositoryInterface $orderRepository)
     {
         $this->entityManager = $entityManager;
+        $this->orderRepository = $orderRepository;
     }
 
     public function getLastYearSalesSummary(ChannelInterface $channel): SalesSummaryInterface
     {
-        $startDate = (new \DateTime('first day of next month last year'))->format('Y/m/d');
-        $endDate = (new \DateTime('last day of this month'))->format('Y/m/d');
-        $channelId = $channel->getId();
-
-        $query = $this->entityManager->getConnection()->query(
-            "SELECT
-                DATE_FORMAT(checkout_completed_at, '%m.%y') AS \"date\",
-                SUM(total) as \"total\"
-            FROM sylius_order
-            WHERE (channel_id = $channelId)
-            AND (checkout_completed_at BETWEEN '$startDate' AND '$endDate')
-            AND (payment_state = 'paid')
-            GROUP BY date;"
-        );
+        $startDate = (new \DateTime('first day of next month last year'));
+        $startDate->setTime(0, 0, 0);
+        $endDate = (new \DateTime('last day of this month'));
+        $endDate->setTime(23, 59, 59);
 
-        $query->execute();
-        $result = $query->fetchAll();
+        $qb = $this->orderRepository->createQueryBuilder('so');
```

Comment:
Our best practice is to reduce usage of abbreviations to the bare minimum (mostly `id` is the only one used) 
```suggestion
        $queryBuilder = $this->orderRepository->createQueryBuilder('so');
```